packages/camera/lib/camera.dart

```diff
@@ -1,355 +1,14 @@
-      );
-    }
-    try {
-      value = value.copyWith(isTakingPicture: true);
-      await _channel.invokeMethod(
-        'takePicture',
-        <String, dynamic>{'textureId': _textureId, 'path': path},
-      );
-      value = value.copyWith(isTakingPicture: false);
-    } on PlatformException catch (e) {
-      value = value.copyWith(isTakingPicture: false);
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Start a video recording and save the file to [path].
-  ///
-  /// A path can for example be obtained using
-  /// [path_provider](https://pub.dartlang.org/packages/path_provider).
-  ///
-  /// The file is written on the flight as the video is being recorded.
-  /// If a file already exists at the provided path an error will be thrown.
-  /// The file can be read as soon as [stopVideoRecording] returns.
-  ///
-  /// Throws a [CameraException] if the capture fails.
-  Future<void> startVideoRecording(String filePath) async {
-    if (!value.isInitialized || _isDisposed) {
-      throw CameraException(
-        'Uninitialized CameraController',
-        'startVideoRecording was called on uninitialized CameraController',
-      );
-    }
-    if (value.isRecordingVideo) {
-      throw CameraException(
-        'A video recording is already started.',
-        'startVideoRecording was called when a recording is already started.',
-      );
-    }
-    try {
-      await _channel.invokeMethod(
-        'startVideoRecording',
-        <String, dynamic>{'textureId': _textureId, 'filePath': filePath},
-      );
-      value = value.copyWith(isRecordingVideo: true);
-    } on PlatformException catch (e) {
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Stop recording.
-  Future<void> stopVideoRecording() async {
-    if (!value.isInitialized || _isDisposed) {
-      throw CameraException(
-        'Uninitialized CameraController',
-        'stopVideoRecording was called on uninitialized CameraController',
-      );
-    }
-    if (!value.isRecordingVideo) {
-      throw CameraException(
-        'No video is recording',
-        'stopVideoRecording was called when no video is recording.',
-      );
-    }
-    try {
-      value = value.copyWith(isRecordingVideo: false);
-      await _channel.invokeMethod(
-        'stopVideoRecording',
-        <String, dynamic>{'textureId': _textureId},
-      );
-    } on PlatformException catch (e) {
-      throw CameraException(e.code, e.message);
-    }
-  }
-
-  /// Releases the resources of this camera.
-  @override
-  Future<void> dispose() async {
-    if (_isDisposed) {
-      return;
-    }
-    _isDisposed = true;
-    super.dispose();
-    if (_creatingCompleter != null) {
-      await _creatingCompleter.future;
-      await _channel.invokeMethod(
-        'dispose',
-        <String, dynamic>{'textureId': _textureId},
-      );
-      await _eventSubscription?.cancel();
-    }
-  }
-}
+part 'src/camera.dart';
```

Comment:
This is just my own preference:
Instead of using part files it is many times possible to reexport the given libraries from the main library.
If you want to share internals between the two sub-files you can move them to `lib/src/`.